docs/src/app/components/AppBar/README.md

```diff
@@ -0,0 +1,7 @@
+## AppBar
+
+App bars are a collection of components placed as a static header for an application.
+It is used for navigation, search branding, and actions.
+An app bar is also referred to as the primary toolbar or action bar for Android.
+
+### Examples
```

Comment:
Examples is inside the readme file, that means every read me MUST end with `###Examples`. shouldn't it be where you make the final markdown FROM this readme file and append this there? also maybe the title should be there too.